### PR TITLE
create privacy.html, set views, urls-web and base file for privacy page

### PR DESCRIPTION
**Pull request by NinaNinchy**
relates to #96

### CODE REVIEW

**Comment by goranche** [comment]
when creating PRs that relate to an issue, please also comment and write something in the lines of the following:

relates to #96

File: web/templates/pages/privacy.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<div>
+		<h3 class="privacy">TrackCat Privacy Policy</h3>
+		<p>When you register for TrackCat we ask for information such as your name, GitHub account and email address. The information we collect is used to improve the content of our web pages and the quality of our service, and is not shared with or sold to other organizations for commercial purposes.</p>
+		<p>TrackCat uses collected information for the following general purposes: identification and authentication, ful acces to our services, contact, and research.</p>
+		<p>It is necessary to share information in order to investigate, prevent, or take action regarding illegal activities, suspected fraud, situations involving potential threats to the physical safety of any person, or as otherwise required by law.</p>
+
+		<p>TrackCat may periodically update this policy. We will notify you about significant changes in the way we treat personal information by sending a notice to the email address specified in your TrackCat account or by placing a prominent notice on our site.</p>
+	</div>
+
+{% endblock content%}
```
Comment (goranche) [inline web/templates/pages/privacy.html]: please add a newline at the end of the file (go to the end of this line and just press enter)

**Comment by NinaNinchy** [comment]
added new line.

**Comment by PettraV** [comment]
Nice wording :) The code looks fine to me :) however you need to add page also in urls file and views file. I do not find those changes, did you include them with pull request?

What about css, did you do any styling? If not, I suggest that you use about site styling, so similar pages just with text look similar. Just an idea, no need need to follow it if you do not like  it :)

**Comment by PettraV** [comment]
pls disregard my above comment, i found the files, sorry :)

**Pull request merged**